exonum/src/sandbox/sandbox_tests_helper.rs

```diff
@@ -159,14 +147,6 @@ impl<'a> ProposeBuilder<'a> {
         self
     }
 
-    pub fn with_duration_since_sandbox_time(
-        mut self,
-        duration_since_sandbox_time: Milliseconds,
-    ) -> Self {
-        self.duration_since_sandbox_time = Some(duration_since_sandbox_time);
-        self
-    }
```

Comment:
Was `with_duration_since_sandbox_time()` removed because this field was set but never actually used or verified anywhere?

Reply:
commit message for  7ba7306 will explain this